includes/Entity/class-item.php

```diff
@@ -282,7 +282,18 @@ public function get_content() {
 	 * @return string
 	 */
 	public function get_response_type() {
-		return $this->response_type ? $this->response_type : 'mention';
+		$response_type = $this->response_type ? $this->response_type : 'mention';
+		// Reclassify short mentions as comments
+		if ( 'mention' === $this->response_type ) {
```

Comment:
```suggestion
		if ( 'mention' === $response_type ) {
```

otherwise it can be empty/null?!?